### PR TITLE
[CostManagement] Release stable version

### PR DESCRIPTION
**Pull request by dvbb**
Link: https://github.com/Azure/azure-sdk-for-net/issues/35656

# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/costmanagement/Azure.ResourceManager.CostManagement/src/autorest.md [hunk 1]
```diff
@@ -89,6 +96,24 @@ rename-mapping:
   Scope: BenefitRecommendationScope
   Grain: BenefitRecommendationUsageGrain
   OperatorType: ComparisonOperatorType
+  DismissAlertPayload.properties.closeTime: -|datetime
```
Comment (live1206) [inline sdk/costmanagement/Azure.ResourceManager.CostManagement/src/autorest.md:99]: consider renaming to CloseOn to align with others

File: sdk/costmanagement/Azure.ResourceManager.CostManagement/src/autorest.md [hunk 2]
```diff
@@ -89,6 +96,24 @@ rename-mapping:
...
+  DismissAlertPayload.properties.statusModificationTime: -|datetime
```
Comment (live1206) [inline sdk/costmanagement/Azure.ResourceManager.CostManagement/src/autorest.md:100]: consider renaming to ModifiedOn

File: sdk/costmanagement/Azure.ResourceManager.CostManagement/src/autorest.md [hunk 3]
```diff
@@ -89,6 +96,24 @@ rename-mapping:
...
+  Alert.properties.modificationTime: -|datetime
```
Comment (live1206) [inline sdk/costmanagement/Azure.ResourceManager.CostManagement/src/autorest.md:101]: consider renaming to ModifiedOn

File: sdk/costmanagement/Azure.ResourceManager.CostManagement/src/autorest.md [hunk 4]
```diff
@@ -89,6 +96,24 @@ rename-mapping:
...
+  Alert.properties.creationTime: -|datetime
```
Comment (live1206) [inline sdk/costmanagement/Azure.ResourceManager.CostManagement/src/autorest.md:102]: consider renaming to CreatedOn

File: sdk/costmanagement/Azure.ResourceManager.CostManagement/src/autorest.md [hunk 5]
```diff
@@ -89,6 +96,24 @@ rename-mapping:
...
+  Alert.properties.statusModificationTime: -|datetime
```
Comment (live1206) [inline sdk/costmanagement/Azure.ResourceManager.CostManagement/src/autorest.md:103]: consider renaming to StatusModifiedOn

File: sdk/costmanagement/Azure.ResourceManager.CostManagement/src/autorest.md [hunk 6]
```diff
@@ -89,6 +96,24 @@ rename-mapping:
...
+  View.properties.scope: -|arm-id
+  ScheduledAction.properties.scope: -|arm-id
+  ScheduledAction.properties.viewId: -|arm-id
+  ExportDeliveryDestination.resourceId: DestinationId|arm-id
```
Comment (live1206) [inline sdk/costmanagement/Azure.ResourceManager.CostManagement/src/autorest.md:107]: consider keeping the name as it is, `Destination` is implied in the class name already, duplicated in the property name.

File: sdk/costmanagement/Azure.ResourceManager.CostManagement/src/autorest.md [hunk 7]
```diff
@@ -89,6 +96,24 @@ rename-mapping:
...
+  KpiProperties.id: KpiId|arm-id
```
Comment (live1206) [inline sdk/costmanagement/Azure.ResourceManager.CostManagement/src/autorest.md:108]: consider keeping the property name as it is

File: sdk/costmanagement/Azure.ResourceManager.CostManagement/src/autorest.md (hunk 1 above)
Comment (dvbb) [reply sdk/costmanagement/Azure.ResourceManager.CostManagement/src/autorest.md:99]: autorest will auto rename all datetime to `FooOn`

File: sdk/costmanagement/Azure.ResourceManager.CostManagement/src/autorest.md (hunk 2 above)
Comment (dvbb) [reply sdk/costmanagement/Azure.ResourceManager.CostManagement/src/autorest.md]: renaming to ModifiedOn will throw 'Sequence contains more than one matching element '
now rename it to `StatusModifiedOn`

**Review by live1206 — APPROVED** [review]

**Pull request merged**